src/main/java/appeng/tile/AEBaseTile.java

```diff
@@ -534,19 +534,7 @@ public void uploadSettings( final SettingsFrom from, final NBTTagCompound compou
 	@Override
 	public void getDrops( final World w, final BlockPos pos, final List<ItemStack> drops )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)